tests/Get-DbaDbCompression.Tests.ps1

```diff
@@ -0,0 +1,63 @@
+$commandname = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandpath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
+    Context "Validate parameters" {
+        $paramCount = 5
+        $defaultParamCount = 11
+        [object[]]$params = (Get-ChildItem function:\Get-DbaDbCompression).Parameters.Keys
+        $knownParameters = 'SqlInstance', 'SqlCredential','Database','ExcludeDatabase','EnableException'
+        It "Should contain our specific parameters" {
+            ( (Compare-Object -ReferenceObject $knownParameters -DifferenceObject $params -IncludeEqual | Where-Object SideIndicator -eq "==").Count ) | Should Be $paramCount
+        }
+        It "Should only contain $paramCount parameters" {
+            $params.Count - $defaultParamCount | Should Be $paramCount
+        }
+    }
+}
+
+Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
+    BeforeAll {
+        Get-DbaProcess -SqlInstance $script:instance1 -Program 'dbatools PowerShell module - dbatools.io' | Stop-DbaProcess -WarningAction SilentlyContinue
```

Comment:
no need to spend time killing all dbatools' processes, get-dbadbcompression isn't doing anything fancy. 
This "forced wanted behaviour" could silence problems in how the connections are handled within our module that'd we wanna know of sooner rather than later. 
Also, we need to be as gentle as possible: this kills every connection done by ANY other dbatools's function (think users running things on another PS process while running tests on their rig). 
If you realy want to make sure to kill "what could get in the way", restrict the sessions to the db you're targeting after creating it, via $server.KillAllProcesses($db). Again, here such a thing is definitely not needed

Reply:
@potatoqualitee are you happy for me to remove this, it was in the example you had sent me for creating a database to test against. Makes sense we wouldn't want to kill everything but didn't want to break something else 😄